lib/stdlib/test/rand_SUITE.erl

```diff
@@ -426,6 +428,55 @@ measure_2(N, State0, Fun) when N > 0 ->
 measure_2(0, _, _) -> ok.
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+-define(LOOP_JUMP, (?LOOP div 1000)).
+
+%% Check if exs64 algorithm generates the proper jump sequence.
+reference_jump(Config) when is_list(Config) ->
+    [reference_jump_1(Alg) || Alg <- algs()],
+    ok.
+
+reference_jump_1(Alg) ->
+    Refval  = reference_jump_val(Alg),
+    Testval = gen_jump(Alg),
+    case Refval =:= Testval of
+        true -> ok;
+        false ->
+	    io:format("Failed: ~p~n",[Alg]),
+	    io:format("Length ~p ~p~n",[length(Refval), length(Testval)]),
+	    io:format("Head ~p ~p~n",[hd(Refval), hd(Testval)]),
+	    ok
+    end.
```

Comment:
To follow the other code use:

```
jump() ->
    Seed = jump(seed_get()),
    _ = seed_put(Seed),
    Seed.
```

event better would be to change seed_put to:

```
-spec seed_put(state()) -> state().
seed_put(Seed) ->
    put(?SEED_DICT, Seed),
    Seed.

jump() ->
    seed_put(jump(seed_get()).

seed(Alg) ->
    seed_put(seed_s(Alg)).

seed(Alg0, S0) ->
    seed_put(seed_s(Alg0, S0)).

```